### PR TITLE
Bump version to v1.9.0

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary
Bump version to v1.9.0

#### Ticket Link
--

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -107,7 +107,7 @@ else
 ifeq ($(BRANCH_EXISTS), 0)
 	@echo "Error: branch ${BRANCH_NAME} already exists"
 else
-	@echo $(NEXT_VER) | grep -Eq ^v[0-9]+\.[0-9]+\.[0-9]+$ || echo "The next version, '$(NEXT_VER)' is not of the form vMAJOR.MINOR.PATCH" && exit 1
+	@echo $(NEXT_VER) | grep -Eq ^v[0-9]+\.[0-9]+\.[0-9]+$ || (echo "The next version, '$(NEXT_VER)' is not of the form vMAJOR.MINOR.PATCH" && exit 1)
```
Comment (agarciamontoro) [inline Makefile:110]: This is needed for the prepare-release subcommand to properly work.

**Review by agnivade — APPROVED** [review]

**Comment by agarciamontoro** [comment]
#580 is merged, so this is now unblocked. I'll repeat the whole performance comparison from 7.10 with current master to verify everything works. Then, we can release v1.9.

**Comment by agnivade** [comment]
> We should maybe reduce the number of users for bounded tests (it's 7.5K right now for Postgres, and 5K for MySQL, that doesn't show this curve).

Yeah good idea

**Comment by agarciamontoro** [comment]
Things looked normal in the test. Similarly to how the bounded test in Postgres showed that degradation at 7K users, the unbounded test showed a lower number of supported users than in [the performance comparison](https://community.mattermost.com/core/pl/rhpkicat9ide3n4zeucgubmouw). While the performance comparison returned 16K-17K max users for Postgres, this test returned around 12K-13K. MySQL showed a similar number than in the performance comparison, also around 12K.

For completeness, here are the Grafana snapshots:

[Postgres bounded](https://snapshots.raintank.io/dashboard/snapshot/KUGFYrr5Yoom1V2bTdoj7Q77gQM6X3eL)
[Postgres unbounded](https://snapshots.raintank.io/dashboard/snapshot/Edp64IUgWT8wl1VOD38BrRmg9VeTmbhz)
[MySQL bounded](https://snapshots.raintank.io/dashboard/snapshot/OottwUTCWFpR9ylA0djn5fkUPJJXrFnx)
[MySQL unbounded](https://snapshots.raintank.io/dashboard/snapshot/HQJWOfafV3ERu1PtSrrzwULAFdWL9fO0)

And the generated graphs:

mysql_bounded_0_avg-api-times
mysql_bounded_0_avg-store-times
mysql_bounded_0_cpu-utilization
mysql_bounded_0_goroutines-in-use
mysql_bounded_0_heap-in-use
mysql_bounded_0_number-of-connected-devices
mysql_bounded_0_p99-api-times
mysql_bounded_0_p99-store-times
mysql_bounded_0_rps
mysql_bounded_0_stack-in-use
mysql_unbounded_0_avg-api-times
mysql_unbounded_0_avg-store-times
mysql_unbounded_0_cpu-utilization
mysql_unbounded_0_goroutines-in-use
mysql_unbounded_0_heap-in-use
mysql_unbounded_0_number-of-connected-devices
mysql_unbounded_0_p99-api-times
mysql_unbounded_0_p99-store-times
mysql_unbounded_0_rps
mysql_unbounded_0_stack-in-use
postgresql_bounded_0_avg-api-times
postgresql_bounded_0_avg-store-times
postgresql_bounded_0_cpu-utilization
postgresql_bounded_0_goroutines-in-use
postgresql_bounded_0_heap-in-use
postgresql_bounded_0_number-of-connected-devices
postgresql_bounded_0_p99-api-times
postgresql_bounded_0_p99-store-times
postgresql_bounded_0_rps
postgresql_bounded_0_stack-in-use
postgresql_unbounded_0_avg-api-times
postgresql_unbounded_0_avg-store-times
postgresql_unbounded_0_cpu-utilization
postgresql_unbounded_0_goroutines-in-use
postgresql_unbounded_0_heap-in-use
postgresql_unbounded_0_number-of-connected-devices
postgresql_unbounded_0_p99-api-times
postgresql_unbounded_0_p99-store-times
postgresql_unbounded_0_rps
postgresql_unbounded_0_stack-in-use

**Pull request merged**